packages/dev/gui/src/2D/controls/inputTextArea.ts

```diff
@@ -0,0 +1,1257 @@
+                }
+            }
+
+            // Avoid the caret during highlighting
+            this._blinkIsEven = this._isTextHighlightOn;
+            this._clickedCoordinateX = null;
+            this._clickedCoordinateY = null;
+        } else {
+            // Standard behavior same as Current line is at least above the initial highlight index
+            this._cursorInfo.relativeStartIndex = 0;
+            this._cursorInfo.currentLineIndex = 0;
+
+            let lineLength = this._lines[this._cursorInfo.currentLineIndex].text.length + this._lines[this._cursorInfo.currentLineIndex].lineEnding.length;
+            let tmpLength = 0;
+
+            while (tmpLength + lineLength <= this._cursorInfo.globalStartIndex) {
+                tmpLength += lineLength;
+
+                if (this._cursorInfo.currentLineIndex < this._lines.length - 1) {
+                    this._cursorInfo.currentLineIndex++;
+                    lineLength = this._lines[this._cursorInfo.currentLineIndex].text.length + this._lines[this._cursorInfo.currentLineIndex].lineEnding.length;
+                }
+            }
+
+            this._cursorInfo.relativeStartIndex = this._cursorInfo.globalStartIndex - tmpLength;
+
+            if (this._highlightCursorInfo.initialStartIndex !== -1 && this._cursorInfo.globalStartIndex >= this._highlightCursorInfo.initialStartIndex) {
+                // Current line is at least below the initial highlight index
+                while (tmpLength + lineLength <= this._cursorInfo.globalEndIndex) {
+                    tmpLength += lineLength;
+
+                    if (this._cursorInfo.currentLineIndex < this._lines.length - 1) {
+                        this._cursorInfo.currentLineIndex++;
+                        lineLength = this._lines[this._cursorInfo.currentLineIndex].text.length + this._lines[this._cursorInfo.currentLineIndex].lineEnding.length;
+                    }
+                }
+
+                this._cursorInfo.relativeEndIndex = this._cursorInfo.globalEndIndex - tmpLength;
+            } else if (!this._isTextHighlightOn) {
+                this._cursorInfo.relativeEndIndex = this._cursorInfo.relativeStartIndex;
+                this._cursorInfo.globalEndIndex = this._cursorInfo.globalStartIndex;
+            }
+        }
+    }
+
+    /**
+     * Update all values of cursor information based on cursorIndex value
+     *
+     * @param offset The index to take care of
+     * @hidden
+     */
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    protected _updateValueFromCursorIndex(offset: number) {
+        // Override to avoid parent behavior during _onPointerMove
+    }
+
+    /**
+     * Select the word immediatly under the cursor on double click
+     *
+     * @param evt Pointer informations of double click
+     * @hidden
+     */
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    protected _processDblClick(evt: PointerInfo) {

```

Comment:
```suggestion
    protected _processDblClick(_evt: PointerInfo) {
```

(or remove the variable altogether :-) )

Reply:
I prefer to keep the parameter, it will be more maintainable after the release (just in case).